client/src/components/NeighborhoodMap.jsx

```diff
@@ -1,34 +1,47 @@
 import React from 'react';
 import Details from './Details';
 import { Map, GoogleApiWrapper } from 'google-maps-react';
-const googleMapApi = require('../../../googleMapConfig');
+import '../../dist/index.css';
+// DO NOT COMMIT ME TO VERSION CONTROL!!!!!
+const GOOGLE_MAPS_API_KEY = 'AIzaSyALnk5uTIhLwMzcanWu3Dspzuh3Gt47C2k';
 
 class NeighborHoodMap extends React.Component {
```

Comment:
No state in this component, so let's refactor to a functional stateless component